mixins/inputMessage.js

```diff
@@ -14,15 +15,17 @@ export default {
   },
 
   shouldShowError () {
-    let errors = this.props.errors || []
+    let propErrors = this.props.errors || []
+    const stateErrors = get(this, 'state.errors') || []
```

Comment:
This is a bit of a doozy that I imagine has grown in complexity over time. The problem is errors may come from either internal state, or prop down.

Prop down happens when a containing component (like CopyInput) recognises it has errors via its local state, but wants to show the error on an inner TextInput.

Local state errors happen when there's local state.

The logic is convoluted, and is ripe for our rewrite after we make Donation Form MVP, by my understanding it's basically this (and this code has a bug that I'm fixing right now on my current issue):

1. If you have errors on props, render the errors, regardless of local state
2. If you have state errors, *and* this.state.hasError is set, render those state errors
3. If you have no state errors, but state.hasError is set, render this.props.errorMessage as an error.